### DISCUSSION TITLE
"Manuelle" Ladesteuerung

### DISCUSSION THREAD

**Discussion by JauntyJosef** [wiggal/GEN24_Ladesteuerung#110] [Q&A] [answered]
Ich habe eine Frage bzw ein Problem mit der Version 0.25.0:

Wenn ich im WebUI die Ladesteuerung auf 100% setze und "Auto" nicht angehakt ist, dann wird trotzdem nicht der Wert, der in meiner charge_priv.ini hinterlegt ist, gesetzt. Ich habe in der charge_priv.ini `10000` für `MaxLadung` hinterlegt. 

Hier der Auszug auf dem LOG:

```
## HTTP-LADESTEUERUNG ##
>>>Parameteränderung durch WebUI-Settings: Ladesteuerung:['logging', 'laden']
aktuellePrognose: 4472
TagesPrognose - BattVollUm: 10347 - 13
Grundlast_Summe für Tag: 1290
aktuellePVProduktion/Watt: 1193
aktuelleEinspeisung/Watt: -2
aktuelleBatteriePower/Watt: -845
GesamtverbrauchHaus/Watt: 350
Akku_Zustand in Prozent: 99.0 %
aktuelleBattKapazität/Watt: 3993
Batteriestatus in Prozent: 60.6 %
LadewertGrund: Prognoseberechnung BatSparFaktor
Bisheriger Ladewert/Watt: 890
Neuer Ladewert/Watt: 928

Änderungen kleiner Schreibgrenze!


DEBUG *************** Berechnung Abzugswert:
DEBUG ##Schleife## Stunden_sum: 0.15, Prognose: 4666, Pro_Ertrag_Tag: 699
DEBUG ##Schleife## Stunden_sum: 1.15, Prognose: 4414, Pro_Ertrag_Tag: 5113
DEBUG ##Schleife## Stunden_sum: 2.15, Prognose: 5234, Pro_Ertrag_Tag: 10347

DEBUG *************** Ladewertmittel LOOP: 0
DEBUG ########### Pro_Akt_fun: 527.25 REST_Akt_Minute: 0.15
DEBUG 2024-10-31 09:00:00 Pro_Akt_fun: 527.25 Prognose_gesamt: 527.25

DEBUG *************** Ladewertmittel LOOP: 1
DEBUG 2024-10-31 10:00:00 Pro_Akt_fun: 4666 Prognose_gesamt: 5193.25

DEBUG *************** Ladewertmittel LOOP: 2
DEBUG ########### Pro_Akt_fun: 3751.9 Akt_Minute: 0.85
DEBUG 2024-10-31 11:00:00 Pro_Akt_fun: 3751.9 Prognose_gesamt: 8945.15

DEBUG 10/31/24 10:51 Aktuelle Prognose - Reservierung: 4472, Batteriekapazität: 3993, aktuellerLadewert: 928

DEBUG <<<<<< Meldungen von Akkuschonung >>>>>>>
DEBUG Akkuschonung_Werte: {'80': '0.2', '90': '0.1', '95': '0.05'}

DEBUG BattVollUm: 13Uhr
DEBUG WRSchreibGrenze_nachUnten: 1500W
DEBUG WRSchreibGrenze_nachOben: 700W
***** ENDE: 2024-10-31 10:51:01 *****
```

Gesetz werden also 928W. Wo habe ich den Fehler?

**Comment by wiggal** [comment]
Ich habe es bei mir nochmal probiert, müsste so aussehen:
```
## HTTP-LADESTEUERUNG ##
>>>Parameteränderung durch WebUI-Settings: Ladesteuerung:['logging', 'laden']
aktuellePrognose:            1892
TagesPrognose - BattVollUm:  0 - 12
Grundlast_Summe für Tag:     0
aktuellePVProduktion/Watt:   1459
aktuelleEinspeisung/Watt:    0
aktuelleBatteriePower/Watt:  -1269
GesamtverbrauchHaus/Watt:    190
Akku_Zustand in Prozent:     94.0 %
aktuelleBattKapazität/Watt:  5343
Batteriestatus in Prozent:   48.6 %
LadewertGrund:  Manuelle Ladesteuerung in PV-Planung ausgewählt.
Bisheriger Ladewert/Watt:    10240
Neuer Ladewert/Watt:         10240

Änderungen kleiner Schreibgrenze!
**In SQLite-Datei gespeichert!**

DEBUG *************** Berechnung Abzugswert: 
DEBUG *************** Ladewertmittel LOOP: 0

```
Wie sieht deine LadeStrg aus, wenn du einen reload machst, müsste so aussehen:
![grafik](https://github.com/user-attachments/assets/f376b92c-e905-460a-bdea-7d81c58b79d8)

Eventuell keine Schreibrechte auf CONFIG/Prog_Steuerung.sqlite?

**Reply by wiggal** [reply]
übrigens bezieht sich die Prozenteinstellungen schon seit längerem auf die Ladekapazität des AKKU und nicht auf MaxLadung.

**Comment by JauntyJosef** [comment]
Vielen Dank!! Die fehlenden Schreibrechte für Pro-Steuerung.sqlite waren das Problem.

**Comment by JauntyJosef** [comment]
Ich habe gerade wieder das gleiche Problem, ohne etwas geändert zu haben:

```
***** BEGINN: 2024-11-01 10:11:04 *****

## HTTP-LADESTEUERUNG ##
>>>Parameteränderung durch WebUI-Settings: Ladesteuerung:['logging', 'laden']
aktuellePrognose: 4732
TagesPrognose - BattVollUm: 10635 - 12
Grundlast_Summe für Tag: 1453
aktuellePVProduktion/Watt: 2052
aktuelleEinspeisung/Watt: 223
aktuelleBatteriePower/Watt: -1484
GesamtverbrauchHaus/Watt: 345
Akku_Zustand in Prozent: 99.0 %
aktuelleBattKapazität/Watt: 5514
Batteriestatus in Prozent: 45.6 %
LadewertGrund: Prognoseberechnung BatSparFaktor
Bisheriger Ladewert/Watt: 1493
Neuer Ladewert/Watt: 1517

Änderungen kleiner Schreibgrenze!


DEBUG *************** Berechnung Abzugswert:
DEBUG ##Schleife## Stunden_sum: 0.817, Prognose: 5325, Pro_Ertrag_Tag: 4348
DEBUG ##Schleife## Stunden_sum: 1.817, Prognose: 6287, Pro_Ertrag_Tag: 10635

DEBUG *************** Ladewertmittel LOOP: 0
DEBUG ########### Pro_Akt_fun: 2988.18 REST_Akt_Minute: 0.817
DEBUG 2024-11-01 09:00:00 Pro_Akt_fun: 2988.18 Prognose_gesamt: 2988.18

DEBUG *************** Ladewertmittel LOOP: 1
DEBUG 2024-11-01 10:00:00 Pro_Akt_fun: 5325 Prognose_gesamt: 8313.18

DEBUG *************** Ladewertmittel LOOP: 2
DEBUG ########### Pro_Akt_fun: 1152.62 Akt_Minute: 0.183
DEBUG 2024-11-01 11:00:00 Pro_Akt_fun: 1152.62 Prognose_gesamt: 9465.8

DEBUG 11/01/24 10:11 Aktuelle Prognose - Reservierung: 4732, Batteriekapazität: 5514, aktuellerLadewert: 1517

DEBUG <<<<<< Meldungen von Akkuschonung >>>>>>>
DEBUG Akkuschonung_Werte: {'80': '0.2', '90': '0.1', '95': '0.05'}

DEBUG BattVollUm: 12Uhr
DEBUG WRSchreibGrenze_nachUnten: 1500W
DEBUG WRSchreibGrenze_nachOben: 700W
***** ENDE: 2024-11-01 10:11:04 *****
```

<img width="894" alt="Bildschirmfoto 2024-11-01 um 10 10 30" src="https://github.com/user-attachments/assets/5b6778d2-db65-4e80-b6e9-3864bcb27d88">


<img width="1191" alt="Bildschirmfoto 2024-11-01 um 10 09 53" src="https://github.com/user-attachments/assets/a3de7536-f873-4893-be0a-679b78f22651">

**Reply by wiggal** [reply]
Hast du das Programm neu installiert?
Was hast du gemacht, bzw. auf was läuft das Programm?
kannst du mal einen `ls -la` vom CONFIG Verzeichnis posten

**Reply by JauntyJosef** [reply]
Nein, nichts neu installiert. Es läuft in einer Debian 12 VM unter Proxmox.
Die "Akkuschonung" wird richtig geschrieben.


```
***** BEGINN: 2024-11-01 12:46:01 *****

## HTTP-LADESTEUERUNG ##
>>>Parameteränderung durch WebUI-Settings: Ladesteuerung:['logging', 'laden']
aktuellePrognose: 6339
TagesPrognose - BattVollUm: 0 - 12
Grundlast_Summe für Tag: 0
aktuellePVProduktion/Watt: 5441
aktuelleEinspeisung/Watt: 596
aktuelleBatteriePower/Watt: 1880
GesamtverbrauchHaus/Watt: 6725
Akku_Zustand in Prozent: 99.0 %
aktuelleBattKapazität/Watt: 141
Batteriestatus in Prozent: 98.6 %
LadewertGrund: Akkuschonung: Ladestand >= 95%, Ladewert = 0.05C
Bisheriger Ladewert/Watt: 512
Neuer Ladewert/Watt: 512

Änderungen kleiner Schreibgrenze!


DEBUG *************** Berechnung Abzugswert:

DEBUG *************** Ladewertmittel LOOP: 0
DEBUG ########### Pro_Akt_fun: 1466.97 REST_Akt_Minute: 0.233
DEBUG 2024-11-01 11:00:00 Pro_Akt_fun: 1466.97 Prognose_gesamt: 1466.97

DEBUG *************** Ladewertmittel LOOP: 1
DEBUG 2024-11-01 12:00:00 Pro_Akt_fun: 6542 Prognose_gesamt: 8008.97

DEBUG *************** Ladewertmittel LOOP: 2
DEBUG ########### Pro_Akt_fun: 4669.77 Akt_Minute: 0.767
DEBUG 2024-11-01 13:00:00 Pro_Akt_fun: 4669.77 Prognose_gesamt: 12678.73

DEBUG 11/01/24 12:46 Aktuelle Prognose - Reservierung: 6339, Batteriekapazität: 141, aktuellerLadewert: 10240

DEBUG <<<<<< Meldungen von Akkuschonung >>>>>>>
DEBUG Akkuschonung_Werte: {'80': '0.2', '90': '0.1', '95': '0.05'}
DEBUG AkkuschonungLadewert-alterLadewert: 0
DEBUG BattStatusProz_Grenze: 90.25
DEBUG AkkuschonungLadewert: 512
DEBUG aktuellerLadewert: 10240

DEBUG BattVollUm: 12Uhr
DEBUG WRSchreibGrenze_nachUnten: 102.4W
DEBUG WRSchreibGrenze_nachOben: 102.4W
***** ENDE: 2024-11-01 12:46:01 *****
```

<img width="507" alt="Bildschirmfoto 2024-11-01 um 12 51 01" src="https://github.com/user-attachments/assets/a6460182-bb7b-4e11-bb2e-20d0507fc346">

Alle Dateien gehören dem Benutzer, der auch das Programm ausführt (dort steht mein Klarname - deswegen geschwärzt). Mit der Datei Prog_Steuerung.sqlite darf sogar (probeweise) jeder alles tun.

**Reply by wiggal** [reply]
ja ok, der Eigentümer ist evtl ausschlaggebend, aber egal. bedeutet das, dass die Rechte irgendwann wieder zurückgesetzt sind, und es mit den Rechten funktioniert?

**Reply by wiggal** [reply]
Hast du den Apache laufen?
Was bringt das Logfile, wenn du PVLadung speicherst?

**Reply by JauntyJosef** [reply]
Ich habe die Rechte gestern so gesetzt und nicht mehr verändert. 
Ja, Apache läuft. 
Welches Logfile meinst du?

**Reply by wiggal** [reply]
`/var/log/apache2/error.log`

Ich vermute mal, dass der Apache nicht unter demselben Eigentümer wie die Skripte laufen.
siehe: https://github.com/wiggal/GEN24_Ladesteuerung/wiki/1)-Installation-GEN24_Ladesteuerung-auf-einem-RaspberryPi

Was sagt folgeder Befehl:
`ps -ef|grep apache`

**Reply by JauntyJosef** [reply]
Also, ich benutze die gleiche Installation, wie auch schon bei einer älteren Version von dir bzw meinem Fork. Dort funktioniert alles problemlos.
Die Apache Konfiguration ist auf den Benutzer, der das Skript ausführt, auch geändert (wie bei deiner Anleitung auf den Benutzer "pi" .. bei mir mein_benutzer)

DIe error.log von Apache bringt beim Aufrufen der Seite folgende Fehlermeldungen:

```
[Sat Nov 02 09:31:19.537649 2024] [mpm_prefork:notice] [pid 471] AH00163: Apache/2.4.59 (Debian) configured -- resuming normal operations
[Sat Nov 02 09:31:19.537700 2024] [core:notice] [pid 471] AH00094: Command line: '/usr/sbin/apache2'
[Sat Nov 02 09:32:08.540791 2024] [php:warn] [pid 477] [client 10.0.0.86:57161] PHP Warning:  Undefined array key "Logging" in /home/GEN24/html/7_tab_Diagram.php on line 61, referer: http://10.0.0.42/
[Sat Nov 02 09:32:08.541647 2024] [php:warn] [pid 474] [client 10.0.0.86:57162] PHP Warning:  Undefined array key "Logging" in /home/GEN24/html/8_tab_Diagram.php on line 61, referer: http://10.0.0.42/
[Sat Nov 02 09:32:08.542037 2024] [php:warn] [pid 474] [client 10.0.0.86:57162] PHP Warning:  Trying to access array offset on value of type null in /home/GEN24/html/8_tab_Diagram.php on line 61, referer: http://10.0.0.42/
[Sat Nov 02 09:32:08.545538 2024] [php:error] [pid 474] [client 10.0.0.86:57162] PHP Fatal error:  Uncaught Exception: Unable to open database: unable to open database file in /home/GEN24/html/8_tab_Diagram.php:107\nStack trace:\n#0 /home/GEN24/html/8_tab_Diagram.php(107): SQLite3->__construct()\n#1 {main}\n  thrown in /home/GEN24/html/8_tab_Diagram.php on line 107, referer: http://10.0.0.42/
[Sat Nov 02 09:32:08.546681 2024] [php:warn] [pid 477] [client 10.0.0.86:57161] PHP Warning:  Trying to access array offset on value of type null in /home/GEN24/html/7_tab_Diagram.php on line 61, referer: http://10.0.0.42/
[Sat Nov 02 09:32:08.549357 2024] [php:error] [pid 477] [client 10.0.0.86:57161] PHP Fatal error:  Uncaught Exception: Unable to open database: unable to open database file in /home/GEN24/html/7_tab_Diagram.php:107\nStack trace:\n#0 /home/GEN24/html/7_tab_Diagram.php(107): SQLite3->__construct()\n#1 {main}\n  thrown in /home/GEN24/html/7_tab_Diagram.php on line 107, referer: http://10.0.0.42/
```

Beim Speichern der PV Ladeleistung kommt in der error.log keine Meldung, aber  in der access.log kommt das:

```
10.0.0.86 - - [02/Nov/2024:09:43:28 +0100] "POST /SQL_speichern.php HTTP/1.1" 200 1024 "http://10.0.0.42/1_tab_LadeSteuerung.php" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1 Safari/605.1.15"
10.0.0.86 - - [02/Nov/2024:09:43:28 +0100] "GET /1_tab_LadeSteuerung.php HTTP/1.1" 200 4608 "http://10.0.0.42/1_tab_LadeSteuerung.php" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1 Safari/605.1.15"
10.0.0.86 - - [02/Nov/2024:09:43:28 +0100] "GET /jquery.min.js HTTP/1.1" 200 30231 "http://10.0.0.42/1_tab_LadeSteuerung.php" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1 Safari/605.1.15"

```

ps -ef | grep apache zeigt folgendes

```
root         471       1  0 09:31 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer      473     471  0 09:31 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer       474     471  0 09:31 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer       475     471  0 09:31 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer      476     471  0 09:31 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer       510     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer       511     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer      512     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer      513     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer     514     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
mein_benutzer      515     471  0 09:32 ?        00:00:00 /usr/sbin/apache2 -k start
root         538     507  0 09:38 pts/0    00:00:00 grep apache
```

**Reply by wiggal** [reply]
OK, der Fehler beim Aufruf kommt dadurch zustande, da du kein Logging machst, und dadurch vermutlich die DB `PV_Daten.sqlite` fehlt, ob das beabsichtigt ist weiß ich nicht.
Folgende Zeile im Logfile zeigt, dass die Daten in die DB gespeichert wurden.
`**In SQLite-Datei gespeichert!**`

Warum die Steuerung nicht in die CONFIG/Prog_Steuerung.sqlite geschrieben bzw. aus ihr gelesen wird, ist mir noch nicht klar.

Folgendes müsste in der DB stehen, wenn du 100% geschrieben hast:
`23:59	Reservierung	ManuelleSteuerung	100	0	`

Vielleicht ist das DB-File auch beschädigt, hast du schon mal versucht es zu löschen (wird beim nächsten Aufruf von http_SymoGen24Controller2.py wieder erzeugt)

**Reply by wiggal** [reply]
irgendwie komm ich gar nicht mehr klar mit deinen Meldungen. 
Du hast keine original Installation oder?
Der Apache kann anscheinend die Pythonconfigs nicht lesen, und schreibt die Steuercodes nicht.
Keine Ahnung warum

**Reply by JauntyJosef** [reply]
Das Löschen der Prog_Steuerung.sqlite hat nichts geholfen. Sie wird neu angelegt, aber es funktioniert trotzdem nicht.
Was meinst die mit original Installation?

**Comment by wiggal** [comment]
`[Sat Nov 02 09:32:08.540791 2024] [php:warn] [pid 477] [client 10.0.0.86:57161] PHP Warning:  Undefined array key "Logging" in /home/GEN24/html/7_tab_Diagram.php on line 61, referer: http://10.0.0.42/`

deutet darauf hin, dass er vom Apacheverzeichnis aus folgende Dateien nicht lesen kann:
../CONFIG/default_priv.ini
../CONFIG/default.ini

Sollte das Apacheverzeichnis html nicht im Skriptverzeichnis liegen, kann er die Dateien nicht finden, und er schreibt die Steuerungsdaten auch irgendwohin, aber nicht in die Scriptverzeichnis/CONFIG/Prog_Steuerung.sqlite.

Das meine ich mit `original Installation` Verzeichnisstruktur genauso wie sie auf Github ist.

**Reply by JauntyJosef** [reply]
Die Verzeichnisstruktur ist aber genau so wie auf Github.
Habe es mit git clone geholt und nichts daran verändern. Eigentlich so wie immer … seltsam. 

Hier die Struktur:

```
.:
CHANGELOG.md  Crontab.log      DynamicPriceCheck.py  html                          LICENSE  README.md                     Solcast_WeatherData.py  SymoGen24Controller2.py  WeatherDataProvider2.py
CONFIG        Crontab.log_weg  FUNCTIONS             http_SymoGen24Controller2.py  pics     Solarprognose_WeatherData.py  start_PythonScript.sh   weatherData.json

./CONFIG:
charge.ini  charge_priv.ini  default.ini  default_priv.ini  dynprice.ini  dynprice_priv.ini  Prog_Steuerung.sqlite  weather.ini  weather_priv.ini  winter.ini

./FUNCTIONS:
DynamicPrice.py  functions.py  GEN24_API.py  httprequest.py  LadewertPruefen.py  PrognoseLadewert.py  __pycache__  SQLall.py  Steuerdaten.py  SymoGen24Connector.py

./FUNCTIONS/__pycache__:
functions.cpython-311.pyc  GEN24_API.cpython-311.pyc  httprequest.cpython-311.pyc  PrognoseLadewert.cpython-311.pyc  SQLall.cpython-311.pyc  Steuerdaten.cpython-311.pyc

./html:
1_Hilfe.html             2_tab_EntladeSteuerung.php  5_tab_Crontab_log.php   7_tab_Diagram.php       9_Hilfe.html       config.php              jquery.min.js      SQL_steuerfunctions.php
1_tab_LadeSteuerung.php  4_Hilfe.html                6_tab_GEN24.php         8_funktion_Diagram.php  9_tab_settigs.php  GEN24Ladesteuerung.png  speichern.php
2_Hilfe.html             4_tab_config_ini.php        7_funktion_Diagram.php  8_tab_Diagram.php       chart.js           index.php               SQL_speichern.php

./pics:
Entladesteuerung.png  Ladesteuerung.png  Ladewertverteilung.png  new2.png  new.png  QZ_Tag.png  Settings.png  Tagesproduktion.png  Tagesverbrauch.png
```

**Reply by wiggal** [reply]
Die Configs kannst du in dem Tab config im Browser lesen bzw neu schreiben z.B. default_priv.ini ?

Und die Prognosedaten werden auch dargestellt?
![grafik](https://github.com/user-attachments/assets/8c02dfe5-8592-4cf7-9878-650315591716)

**Reply by JauntyJosef** [reply]
Ja, kann ich alles machen.

In die Prog_Steuerung.sqlite dürfte allerdings der richtige Wert geschrieben werden. Ich habe auf 70% gestellt, gespeichert und dann in die sqlite Datei gesehen. Die letzte Zeile lautet:
`23:59|Reservierung|ManuelleSteuerung|70|0|`

Beim Wechselrichter kommt dies dann aber anscheinend nicht an. Und ja, der Wechselrichter ist richtig konfiguriert, denn die Akkuschonung funktioniert.

**Reply by wiggal** [reply]
Ich glaub ich habs, 
so muss der Eintrag in deiner charge_priv.ini lauten:

```
[Reservierung]
; Hier kann die PV-Reserierung (z.B. für ein E-Auto nachmittags) eingeschaltet werden (1=ein, 0=aus)
PV_Reservierung_steuern = 1
```

**Reply by wiggal** [reply]
Ist aber keine Erklärung, für die Fehlermeldung:

[Sat Nov 02 09:32:08.540791 2024] [php:warn] [pid 477] [client 10.0.0.86:57161] PHP Warning:  Undefined array key "Logging" in /home/GEN24/html/7_tab_Diagram.php on line 61, referer: http://10.0.0.42/

kannst du mal den Inhalt der beiden Dateien posten:
CONFIG/default_priv.ini
CONFIG/default.ini

**Reply by JauntyJosef** [reply, marked as answer]
> Ich glaub ich habs, so muss der Eintrag in deiner charge_priv.ini lauten:
> 
> ```
> [Reservierung]
> ; Hier kann die PV-Reserierung (z.B. für ein E-Auto nachmittags) eingeschaltet werden (1=ein, 0=aus)
> PV_Reservierung_steuern = 1
> ```

Jetzt klappt es !!!

**Reply by JauntyJosef** [reply]
> Ist aber keine Erklärung, für die Fehlermeldung:
> 
> [Sat Nov 02 09:32:08.540791 2024] [php:warn] [pid 477] [client 10.0.0.86:57161] PHP Warning: Undefined array key "Logging" in /home/GEN24/html/7_tab_Diagram.php on line 61, referer: http://10.0.0.42/
> 
> kannst du mal den Inhalt der beiden Dateien posten: CONFIG/default_priv.ini CONFIG/default.ini

Ich habe jetzt den letzen Eintrag in der Apache error.log um 13:15 Uhr. Aktuell kommen dort keine neuen Eintragungen, wenn ich die Seite aufrufe. Was genau ich da aber getan habe, kann ich nicht mehr nachvollziehen. Eventuell das Löschen der sqlite Datei oder das Aktivieren des Loggings in der Config.
